src/testUtils.tsx

```diff
@@ -18,13 +20,18 @@ const customRender = (
       <Router history={history}>
         <AppProvider theme={theme} i18n={enTranslations}>
           <RestfulProvider base={process.env.REACT_APP_API_URL ?? ""}>
-            {children}
+            <Suspense fallback={<Spinner />}>
```

Comment:
Ah, maybe you do need to set `suspense` to false from earlier, but I think only if you grab your translations via ajax.

Reply:
the tests will fail without the <Suspense>

Reply:
The tests will fail without the `<Suspense>` tag.